lib/slax_web/websockets/issue.ex

```diff
@@ -10,50 +10,38 @@ defmodule SlaxWeb.Issue do
   def handle_event(%{"bot_id" => bot_id}) when not is_nil(bot_id), do: nil
 
   def handle_event(%{"thread_ts" => ts, "channel" => channel, "text" => text, "type" => "message"}) do
-    with true <- String.match?(text, ~r{([\S-]+/)?([\S-]+)?(#[0-9]+)}),
-         {:ok, issue} <- load_issue(text) do
-      repo_and_issue =
-        Regex.replace(~r".*/repos/(\S+)/(\S+)/issues/(\d+)$", issue["url"], "\\1/\\2/\\3")
-
-      Slack.post_message_to_thread(%{
-        text:
-          "<#{issue["html_url"]}|#{repo_and_issue}>: #{issue["title"]} #{labels_for_issue(issue)}",
-        channel: channel,
-        thread_ts: ts
-      })
+    with issues when issues != [] <- Regex.scan(~r{([\w-]+/)?([\w-]+)?(#[0-9]+)}, text) do
+      reply =
+        issues
```

Comment:
should each issue be it's own message so the links unfurl? or does it work when they are all together still?

Reply:
I have the links set to not unfurl since that's how it responds now, plus I think if they unfurl it would take up way too much space